tests/conftest.py

```diff
@@ -212,5 +213,75 @@ def layout_array_north_instance(io_handler, db_config):
 
 
 @pytest.fixture
-def corsika_telescope_data_dict(layout_array_north_instance):
-    return layout_array_north_instance._from_corsika_file_to_dict()
+def layout_array_south_instance(io_handler, db_config):
+    return LayoutArray(site="South", mongo_db_config=db_config, name="test_layout")
+
+
+@pytest.fixture
+def manual_corsika_dict_north():
+    return {
+        "corsika_sphere_radius": {
+            "LST": 12.5 * u.m,
+            "MST": 9.15 * u.m,
+            "SCT": 7.15 * u.m,
```

Comment:
Indeed. I removed it now

Reply:
Actually, the way `_from_corsika_file_to_dict` is constructed now, it always returns the sizes of all the telescope types in the corsika_parameters.yml file, regardless of the site you provide. I think this is not problem and would prefer to continue with that, which means returning the SCT and SST to the list above. If you prefer, we would have to adapt `_from_corsika_file_to_dict` to provide different dictionaries depending on the site

Reply:
No, I don't think we need to modify the code, just these dictionaries. If you anyway always read `corsika_parameters.yml` though, then why do you even define these dictionaries here?

Reply:
because sometimes we are not interested in testing reading the file but test functions that use the dictionary, hence, in order to avoid using the read function inside other tests I just manually defined the dictionary here

Reply:
OK!